decidim-meetings/spec/system/participatory_processes_view_hooks_spec.rb

```diff
@@ -17,10 +17,10 @@
       visit resource_locator(participatory_process).path
       expect(page).to have_css(".past_meetings .card--list__item", count: 3)
 
-      expect(page).not_to have_content(/#{translated(moderated_meeting.title)}/i)
+      expect(page).not_to have_content(translated(moderated_meeting.title))
```

Comment:
Consistency with develop
```suggestion
      expect(page).not_to have_content(translated(/#{moderated_meeting.title}/i))
```